### PR TITLE
Breaking issue with GHI 2.1.19 on Win10.

### PR DESCRIPTION
**Pull request by Kruithne**
Hey!

The latest version of GHI (2.1.19) is causing strange issues with users on Windows 10 who are now unable to load up their WoW while GHI is active, remove the add-on and other strange bugs.

I originally suspected that it was something to do with TRP3 Extended since they updated their add-on communication protocol 3 hours before the issue was reported, but after speaking with Ellypse (TRP developer) it was just a co-incidence.

I checked over the commits for the latest version multiple times and had some other people do the same, but there was obviously no signs of breaking code. Started to write the issue off as HDD failure but then even more people started reporting the issue on both EU and US forums, as well as on Twitter.

After getting the developers at Curse involved, it seems the ZIP file that you've uploaded for GHI 2.1.19 is corrupted in some way and they've suggested that you re-upload the latest version of GHI to hopefully fix the issue.

TL;DR: Latest version of GHI is corrupted on Curse and breaks Win10 users, please re-upload a new version ASAP.

P.S Couldn't find an e-mail address or decent method of direct communication, and issues are disabled for the project here, hence why this is a bogus PR.

### CODE REVIEW

**Comment by Kruithne** [comment]
The guys are Curse also suggested using a different tool to produce the ZIP file, rather than the one you are currently using. I'm not sure if the issue is with the tool you used, or a botched upload, but that's what they've suggested either way.

**Comment by Kruithne** [comment]
Just to clarify, although the latest version does appear to be broken in some way, it is only Windows 10 users that are having issues. Other versions of windows as well as Mac OSX have no problems with the package being broken.

**Comment by Pilus** [comment]
Thanks for the info @Kruithne. I have started investigating the issue and I can indeed reproduce that issue and other weird situations locally. It seems that copying the GHI files around or packing them takes many times longer than normal. It seems to be the lua files that is slowing the copying down. This is true for GHI/GHM files directly from the repo as well, including them just downloaded as a zip.

Would you be able to verify that version 2.1.18 even works? I am observing the same issues unpacking and copying those files.

P.s. feel free to contact me on pilus@pilus.info as a direct form of communication.

**Comment by Kruithne** [comment]
I'm not able to reproduce the bug on my machine, but I did pass a clean copy of 2.1.18 onto two of the people affected by the problem, and they both reported long extraction times and problems moving the files, so it appears to be happening there too as you said.

It seems Google Chrome struggles with the files and isn't properly understanding how to download them, causing the downloader module to break and become unresponsive when downloading other things until restarted. So the problem definitely occurs even when the files are zipped.

**Comment by Kruithne** [comment]
Despite the fact that it effects 2.1.18 as well, the problem only seems to have started happening today. I've been keeping an eye on the forums and it seems more and more people are starting to report it.

I've kept the situation up-to-date here: http://eu.battle.net/forums/en/wow/topic/17613694209

I've also redirected people to the EU forums for that thread in three places:
http://us.battle.net/forums/en/wow/topic/20749698126
http://us.battle.net/forums/en/wow/topic/20749688238
http://us.battle.net/forums/en/wow/topic/20749728367

**Comment by Pilus** [comment]
Thanks for keeping people updated. I think I have been able to isolate the issue or at least a workaround. It seemed that it was the lines of   --==========  characters that I have as file header in each lua file which triggers the issue. I have removed them from all files and now the files seems to be copyable as normal. I will test it out in wow later today when I am at a computer with wow on and then release an update.

**Comment by Pilus** [comment]
It seems to have worked fine. I have released a new version of GHI.

**Comment by Kruithne** [comment]
One of the people on Twitter who was having an issue has informed me that they have updated but are still experiencing the issue. Does GHM suffer from the same thing perhaps?

**Comment by Pilus** [comment]
GHM have been updated in the same way. Have the user tried to delete the existing GHI and GHM folders before installing the new one?
